spaces/S000074/properties/P000117.md

```diff
@@ -0,0 +1,11 @@
+---
+space: S000074
+property: P000117
+value: true
+---
+
+A $\sigma$-locally finite network can be constructed out of a countable basis $\mathcal B$ of $\mathbb R\times(0,+\infty)$ (countable union of families consisting of single sets)
+and a family $\mathcal E=\{ \{(x,0)\}: x\in\mathbb R\}$.
+Each point admits a neighbourhood intersecting at most one element of any of the families.
+
+Since every open set in $X$ is a union of an open subset of the Euclidean half-plane as some subset of the edge, the family described above is indeed a network.
```

Comment:
```suggestion
A $\sigma$-locally finite network can be constructed as the union of a countable basis $\mathcal B$ of $\mathbb R\times(0,+\infty)$ (countable union of families, each consisting of a single set)
and a family $\mathcal E=\{ \{(x,0)\}: x\in\mathbb R\}$.
Each point admits a neighbourhood intersecting at most one element of any of the families.

Since every open set in $X$ is a union of an open subset of the Euclidean open half-plane and some subset of the edge, the family described above is indeed a network.
```